play.pokemonshowdown.com/src/battle-log.ts

```diff
@@ -17,6 +17,7 @@ import type { Battle } from './battle';
 import type { BattleScene } from './battle-animations';
 import { Dex, Teams, toID, toRoomid, toUserid, type ID } from './battle-dex';
 import { BattleTextParser, type Args, type KWArgs } from './battle-text-parser';
+import { Net } from './client-connection'; // optional
```

Comment:
Oh, I guess you didn't know. In client, import/export are entirely ignored. That's why `if (window.Net)` works in the first place.